### PR TITLE
minor update -new /v4/product/info/attributes endpoint +test

### PR DESCRIPTION
**Pull request by benice2me11**
minor update -new /v4/product/info/attributes endpoint +test

as i described in issues thread

### CODE REVIEW

**Comment by diPhantxm** [comment]
Hi!
Thank you for your contribution to this library. I'm a little bit busy right now, I will have a look soon

File: ozon/products.go [hunk 1]
```diff
@@ -1558,6 +1558,92 @@ func (c Products) GetDescriptionOfProduct(ctx context.Context, params *GetDescri
 	return resp, nil
 }
 
+type GetDescriptionOfProductFilterV4 struct {
```
Comment (diPhantxm) [inline ozon/products.go:1561]: Let's remove any mention of version in naming

File: ozon/products.go [hunk 2]
```diff
@@ -1558,6 +1558,92 @@ func (c Products) GetDescriptionOfProduct(ctx context.Context, params *GetDescri
...
+	Depth                 int32  `json:"depth"`
+	Width                 int32  `json:"width"`
+	DimensionUnit         string `json:"dimension_unit"`
+	Weight                int32  `json:"weight"`
+	WeightUnit            string `json:"weight_unit"`
+	DescriptionCategoryId int64  `json:"description_category_id"`
+	TypeId                int64  `json:"type_id"`
+	PrimaryImage          string `json:"primary_image"`
+
+	// Доп. структура для "model_info"
```
Comment (diPhantxm) [inline ozon/products.go:1599]: Let's use only english in code

**Review by diPhantxm — CHANGES_REQUESTED** [review]

File: ozon/products.go (hunk 1 above)
Comment (benice2me11) [reply ozon/products.go]: done

File: ozon/products.go (hunk 2 above)
Comment (benice2me11) [reply ozon/products.go]: done

**Review by diPhantxm — APPROVED** [review]

**Comment by diPhantxm** [comment]
LGTM
Thank you for your contribution. I will be pleased to see more :)

**Pull request merged**